apps/parsley/cypress/integration/sectioning.ts

```diff
@@ -0,0 +1,80 @@
+describe("Sectioning", () => {
+  beforeEach(() => {
+    cy.visit(`${logLink}?shareLine=0`);
+    cy.clickToggle("sections-toggle", true, "log-viewing");
+  });
+  it("Toggling the sections options displays and hides sections", () => {
```

Comment:
Super nit but I do appreciate when there are newlines between test blocks for scannability